doc/images/association-rules-tags.txt

```diff
@@ -0,0 +1,4 @@
+0	252	34
+1	252	138
+2	252	318
```

Comment:
I understand what these are (it's self-evident), but how do they "work"? How are they used?